tests/src/integration/message-hub/manifest.yaml

```diff
@@ -0,0 +1,29 @@
+packages:
+    data-processing-with-messagehub:
+        dependencies:
+            wskdeployMessageHub:
+                location: /whisk.system/messaging
+                inputs:
+                    user: $MESSAGEHUB_USER
+                    password: $MESSAGEHUB_PASSWORD
+                    kafka_admin_url: $MESSAGEHUB_ADMIN_HOST
```

Comment:
How can we pick up the admin host in Travis?

Reply:
I am setting it in the test file `message-hub_test.go`

Reply:
https://github.com/apache/incubator-openwhisk-wskdeploy/pull/528/files#diff-189c157cd9e283389d43479ba4b0cd86R39